eng/pipelines/templates/variables/globals.yml

```diff
@@ -1,5 +1,5 @@
 variables:
-  DotNetCoreSDKVersion: '3.0.100-preview5-011568'
+  DotNetCoreSDKVersion: '3.0.100-preview7-012821'
```

Comment:
I'm OK with moving to preview7 assuming the builds work, but please update the docs to call this out as well https://github.com/Azure/azure-sdk-for-net/blob/master/CONTRIBUTING.md. 

FYI @chidozieononiwu